src/Native/build-native.cmd

```diff
@@ -50,9 +50,9 @@ if "%VisualStudioVersion%"=="14.0" (
 )
 
 :MissingVersion
-:: Can't find VS 2013+
-echo Error: Visual Studio 2015 required
-echo        Please see https://github.com/dotnet/corefx/blob/master/Documentation/project-docs/developer-guide.md for build instructions.
+:: Can't find VS 2015
+echo Error: Visual Studio 2015 required  
```

Comment:
FWIW I think we should actually fall back to using the .NET Core msbuild if we don't detect msbuild on the path. We already have everything we need in the Tools directory to do that.